core/templates/dev/head/pages/library/library.html

```diff
@@ -173,7 +173,7 @@ <h2 ng-class="{'active': activeGroupIndex === $index}" class="oppia-library-grou
 
               <div class="oppia-library-carousel"
                    ng-swipe-right="decrementLeftmostCardIndex($index)"
-                   ng-swipe-left="incrementLeftmostCardIndex($index,(group.activity_summary_dicts.length - tileDisplayCount) <= leftmostCardIndices[$index])">
+                   ng-swipe-left="incrementLeftmostCardIndex($index,group.activity_summary_dicts.length,tileDisplayCount)">
```

Comment:
Hmm, this wasn't what I had in mind, is it possible to reference `group` and `tileDisplayCount` in the js file?

Reply:
Hi @kevinlee12 , What exactly do you have in mind? Can you please help me out a little. Thanks :)

Reply:
I have referenced `tileDisplayCount` in the js file.

Reply:
Looks like you've got it :smile: